wsapi.go

```diff
@@ -267,9 +267,16 @@ type helloOp struct {
 // FailedHeartbeatAcks is the Number of heartbeat intervals to wait until forcing a connection restart.
 const FailedHeartbeatAcks time.Duration = 5 * time.Millisecond
 
+// latency returns the latency between heartbeat acknowledgement and heartbeat send
+func (s *Session) Latency() int64 {
```

Comment:
Can this method simply return the time.Time?

Reply:
It could, but I just thought returning milliseconds would make things easier.

Reply:
Returning time.Time is a little more standard and explicit, I don't need to guess at the unit of the value, I didn't realise you were returning millis for example.

Reply:
Ah, I see where you're coming from now. So just return ``s.LastHeartbeatAck.Sub(s.LastHeartbeatSent)``?

Reply:
Yeah I think that would be great.

Reply:
It's gonna have to return ``time.Duration``, since ``Sub()`` returns a Duration